mix.exs

```diff
@@ -41,13 +41,15 @@ defmodule Community.Mixfile do
   defp deps do
     [
       {:cowboy, "~> 1.0"},
+      {:ecto, "~> 2.0.0-rc", [override: true, optional: false, hex: :ecto, manager: :mix]},
```

Comment:
Ok that’s good to know :) Thanks!

> On Apr 29, 2016, at 1:21 PM, Jason Draper <notifications@github.com> wrote:
> 
> In mix.exs <https://github.com/drapergeek/community/pull/26#discussion_r61614494>:
> 
> > @@ -41,13 +41,15 @@ defmodule Community.Mixfile do
> >    defp deps do
> >      [
> >        {:cowboy, "~> 1.0"},
> > +      {:ecto, "~> 2.0.0-rc", [override: true, optional: false, hex: :ecto, manager: :mix]},
> This was recommended by mix because of conflicting dependencies among the packages.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/drapergeek/community/pull/26/files/2cd7406faebf0f015105398feb0e633ebba1bbe6#r61614494>